openfe/setup/ligand_network_planning.py

```diff
@@ -160,7 +167,7 @@ def generate_maximal_network(
     ----------
     ligands : Iterable[SmallMoleculeComponent]
       the ligands to include in the LigandNetwork
-    mappers : AtomMapper or Iterable[AtomMapper]
+    mapper : AtomMapper or Iterable[AtomMapper]
```

Comment:
This docstring rename doesn't match the current signature (note that we can't rename from mappers to mapper without causing an API break).

Reply:
@atravitz you marked this as resolved but the docstring still doesn't match - was this intentional?